### DISCUSSION TITLE
DucoBox HTTP node commands

### DISCUSSION THREAD

**Discussion by MarkRoks** [TheWalkingDadNL/ducoboxHTTP#2] [Ideas]
Hello,

Does anyone happen to have the commands that can be applied to the DucoBoxHTTP node within Node-Red? I am familiar with the commands that use the API. But these are not recognized when I send them to the DucoBoxSend.

I like to hear it.

Regards,

**Comment by TheWalkingDadNL** [comment]
Currently the following commando's can be used as input for the node ducoboxSend:


```
- msg.CO2Setpoint

    Setpoint for your CO2. If ducobox measures more CO2 then the setpoint it will increase flow

- msg.RHSetpoint

    Setpoint for your humidity. If ducobox measures more humidity then the setpoint it will increase flow

- msg.Operation_State

    State of ducobox. Available states are: AUTO, AUT1, AUT2, AUT3, MAN1, MAN2, MAN3, EMPT, CNT1, CNT2, CNT3. Default setting: AUTO
``` 

Those commando's were enough for me. What commando's are you still missing?

**Reply by MarkRoks** [reply]
Hello,

Obviously, this is exactly what I wanted to know! The composition of the message was not clear to me. The values ​​that are sent were already known and apparently correspond to those I already had.

What could still be sent is a value with which all auto/man/cnt positions are overruled. So the actual desired fan speed. Known to me as "setoverrule" as a command.

I like to hear it.
